CHANGELOG.md

```diff
@@ -9,6 +9,11 @@
 * Improve performance of `line_length` rule.  
   [Marcelo Fabri](https://github.com/marcelofabri)
 
+* Add `closure_body_length` rule to enforce the maximum number
```

Comment:
Can you please mention that the rule is opt-in?

Reply:
Of course :-) Sorry about that.